runtime/v2/shim/util_windows.go

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
I think you meant to remove this file since you moved this under shim with your latest rebase. It no longer makes sense for Windows at all correct? The whole `shim` folder is `// +build !windows` in theory.

Reply:
Actually I think we will have to make this package work with windows.  It would require named pipe support but other than that, we can start removing the build tags and get it working.

Reply:
I'll remove the build tag and get this package refactored now